include/cuco/static_set.cuh

```diff
@@ -43,8 +43,28 @@ namespace experimental {
 /**
  * @brief A GPU-accelerated, unordered, associative container of unique keys.
  *
- * @note cuCollections data stuctures always place the slot keys on the left-hand
- * side when invoking the key comparison predicate.
+ * The `static_set` supports two types of operations:
+ * - Host-side "bulk" operations
+ * - Device-side "singular" operations
+ *
+ * The host-side bulk operations include `insert`, `contains`, etc. These APIs should be used when
+ * there are a large number of keys to modify or lookup. For example, given a range of keys
```

Comment:
I somehow can't do suggestions here. Can you replace "keys" with elements in effort to make these docs more generic when we copy&paste them around?

Reply:
Maps and sets are different in the way that sets don't handle associative values. This is the place I'd like to highlight the difference instead of having convenient share documentation

Reply:
Good point. Haven't thought about that.